build/download_docker_compose_binary.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+PLATFORM=$1
+ARCH=$2
+DOCKER_COMPOSE_VERSION=$3
+
+if [ "${PLATFORM}" == 'win' ]; then
```

Comment:
why do we need to handle windows? the gruntfile calls this script only with mac and Linux.